manage/manage.css

```diff
@@ -746,47 +763,40 @@ input[id^="manage.newUI"] {
   display: none !important;
 }
 
-fieldset {
-  border-width: 1px;
-  border-radius: 6px;
-  margin: 1em 0;
-  min-width: 0;
-  max-width: 250px;
-}
-
-fieldset > input,
-fieldset > label {
+#filters input,
+#filters label {
   display: flex;
   align-items: center;
 }
 
-#header fieldset > label {
+#filters label {
   padding-left: 20px;
 }
 
-#header fieldset > label input[type="checkbox"]:not(.slider),
-#header fieldset > label input[type="checkbox"]:not(.slider):checked + .svg-icon.checked{
+#filters label input[type="checkbox"]:not(.slider),
+#filters label input[type="checkbox"]:not(.slider):checked + .svg-icon.checked{
   left: 4px;
 }
 
 #filters {
   border: 1px solid transparent;
 }
 
-#filters.active {
-  background-color: darkcyan;
-  border-color: darkcyan;
+.active #filters-stats {
+  background-color: hsla(180, 100%, 20%, 1);
```

Comment:
don't change the color, there was no need, also font-weight from h2 should be reverted, vertical paddings reduced, horizontal paddings enlarged, rounding removed (we're using it only on "tags"):

```css
.active #filters-stats {
  background-color: darkcyan;
  border-color: darkcyan;
  color: white;
  font-size: 0.7rem;
  font-weight: normal;
  padding: 2px 5px;
  position: relative;
  top: -2px;
}
```

Reply:
I made it match the usercss labels. Having it darkcyan kinda looks weird.

Reply:
No, what's weird is a UI element having the same color as something completely unrelated.

Reply:
when you use the same color it looks like a standard accent color, but in that case why usercss label uses the accent color? why is it accented? it's just a type/tag, there's nothing particularly important to the overall UI about that.